lib/pf/profile/filter/all.pm

```diff
@@ -0,0 +1,59 @@
+package pf::profile::filter::profile::filter::all;
```

Comment:
Now next question, I guess the code doesn't work with that mistake. So is this PR already tested or it needs testing before it's merged ?